cg/store/crud/read.py

```diff
@@ -1729,23 +1730,24 @@ def get_all_pools_to_deliver(self) -> list[Pool]:
         )
         return records.all()
 
-    def get_orders_by_workflow(
-        self, workflow: str | None = None, limit: int | None = None
-    ) -> list[Order]:
-        """Returns a list of entries in Order. The output is filtered on workflow and limited, if given."""
-        orders: Query = self._get_query(table=Order)
-        order_filter_functions: list[Callable] = [OrderFilter.FILTER_ORDERS_BY_WORKFLOW]
-        orders: Query = apply_order_filters(
-            orders=orders, filter_functions=order_filter_functions, workflow=workflow
-        )
-        return orders.limit(limit).all()
+    def get_orders(self, workflow: Workflow | None = None, limit: int | None = None) -> list[Order]:
```

Comment:
I remember wanting to name it like this, but it was pointed out that if filtering occurs, convention is that the name reflects it. Might be worth considering.

Reply:
I think that is an implementation detail that the caller should not care about.